netcore-webview-test/MainWindow.xaml.cs

```diff
@@ -44,21 +44,23 @@ async void MessageReceived(object sender, CoreWebView2WebMessageReceivedEventArg
 
         private class PsMessage
         {
-            public int msgId { get; set; }
-            public string action { get; set; }
-            public string data { get; set; }
+            public int msgId;
```

Comment:
I wasn't aware that field support was already added :)

Reply:
yeah, me neither. I did the same thing before your PR, just went with the properties. Good to know though.
This is just a PoC, doesn't have to be prod quality, and we'll probably let SignalR do the low level work. But it's good to see the performance we can get with raw websocket.

Reply:
We should still keep ws on the table. It's more accessible to app developers than signalr.